src/message/html/js.js

```diff
@@ -63,28 +63,20 @@ window.addEventListener('scroll', () => {
 });
 
 document.body.addEventListener('click', e => {
-  alert('click');
-  sendMessage({
-    type: 'click',
-    target: e.traget,
-    targetNodeName: e.target.nodeName,
-    targetClassName: e.target.className,
-    matchez: e.target.matches('a[target="_blank"] > img'),
-  });
-
   if (e.target.matches('.avatar-img')) {
     sendMessage({
       type: 'avatar',
       fromEmail: e.target.getAttribute('data-email'),
     });
+    return;
   }
 
-  if (e.target.matches('.header')) {
+  if (e.target.className === 'title-text' || e.target.className === 'private-header header' || e.target.className === 'topic-header header' || e.target.className === 'stream-text') {
     sendMessage({
       type: 'narrow',
       narrow: e.target.getAttribute('data-narrow'),
-      id: e.target.getAttribute('data-id'),
     });
+    return;
```

Comment:
This is working as expected. No need to add a `return` keyword.
Multiple handlers can intersect.